portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String firstName = request.getParameter("firstName");
+    String lastName = request.getParameter("lastName");
+    String email = request.getParameter("email");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("firstname: " + firstName);
```

Comment:
Generally, you wouldn’t submit debugging logs to your final product. I’m not sure how frontend works though and if these are okay style there. I would just Google it!

Reply:
Noted, when I was coding it I was thinking the same thing. It seemed to me that they are pretty much accomplishing the same thing.